pkg/plugins/golang/v3/scaffolds/api.go

```diff
@@ -99,6 +99,7 @@ func (s *apiScaffolder) Scaffold() error {
 		if err := scaffold.Execute(
 			&controllers.SuiteTest{Force: s.force},
 			&controllers.Controller{ControllerRuntimeVersion: ControllerRuntimeVersion, Force: s.force},
+			&controllers.ControllerTest{Force: s.force, DoAPI: doAPI},
```

Comment:
Can you please remove it from here?
the plugin go/v3 is deprecated so we should not perform changes under this one
Any change must be applied only in go/v4

Reply:
Ok.